### ISSUE TITLE
NVDA ignoring line breaks in PDF documents with left-to-right, top-to-bottom reading order

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#1216]
**Reported by parham on 2010-11-21 15:09**
When accessibility options in Adobe Acrobat Reader are set to "Read order: left to right, top to bottom", NVDA ignores line breaks completely.

Steps to reproduce:
1. Open a PDF document in the latest version of Acrobat Reader.
2. Look at the first page with the reading order set to "infer from document". You will see some line breaks.
3. Now, set the reading order to "left to right, top to bottom". For this you should go to the document menu, accessibility options, and click next until you get to the reading order radio buttons. Change it to the item I said above, and click next until you get to the done button.
4. Go to the next page and return back to the first page. The page is now all in continuous lines.

**Comment by leonardder** [comment]
@camlorn: Can you still reproduce this?

CC @JCSTeh

**Comment by jcsteh** [comment]
I think Acrobat completely ignores any tagging when you switch to any reading order other than "infer from document". Thus, the idea of block elements (which NVDA relies on to break lines) aren't present. This should be fixed by the fix for #7275.

**Comment by camlorn** [comment]
I have no idea.  I'll try if it's needed.

**Comment by Adriani90** [comment]
I am testing with NVDA 2019.1.1 and Adobe Acrobat Reader DC 19.010.20099. I cannot reproduce this issue. I don't notice any difference between those two read orders. @camlorn, @parham can you confirm? Or are you still having this problem?

**Comment by camlorn** [comment]
I don't even remember having this issue in the first place at this point, and haven't touched pdfs in a very long time, so: no, but not necessarily because it's gone.

**Comment by Neurrone** [comment]
Duplicate of #7275.

This is definitely still happening, and I was able to reproduce it with source code of a pdf ebook.

**Comment by Adriani90** [comment]
Closing as duplicate. thanks for the comment.
